### PR TITLE
refactor: refactor pdf segmenter

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Review by bwanglzu — CHANGES_REQUESTED** [review]
same make `crafter` instance as fixture?

File: jinahub/segmenters/PDFSegmenter/pdf_segmenter.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 class PDFSegmenter(Executor):
     """
-    :class:`PDFCrafter` Extracts data (text and images) from PDF files.
+    :class:`PDFSegmenter` Extracts data (text and images) from PDF files.
```
[the remark this answers is not in the document]
Comment (JoanFM) [reply jinahub/segmenters/PDFSegmenter/pdf_segmenter.py:16]: sure

**Comment by JoanFM** [comment]
@bwanglzu please review again

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**